website/docs/reference/notifications.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Notifications
+---
+
+:::info Availability
+
+Notifications were introduced in Unleash 4.22.0 for pro and enteprise customers.
+
+:::
+
+Unleash's notifications give you updates when certain events occur in projects that you are part of. The notifications are accessible from the notifications button in the navigation section of the admin UI.
```

Comment:
Are notifications available via the API or are they a UI-only thing today? Are the OpenAPI specs up to date with this already?

Reply:
OpenAPI specs are up to date even though this API is mostly for the UI. If someone wants to use the API though it's there and fully documented.